internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -767,6 +777,20 @@ components:
             $ref: '#/components/schemas/Repository'
         ostree:
           $ref: '#/components/schemas/OSTree'
+        upload_target:
```

Comment:
Is it possible that we may want to upload to more than one of these? If so then it might be good to just make this a list now instead of dealing with singular/plural problems later :)

Reply:
I think that would require `upload_options` to also be a list, which isn't a compatible change.  You're not wrong though, I'm just not sure how to tackle this.  I'm open to ideas.

Reply:
Ah, I see. Then it would need to be a list of objects with both items. Not something that can be done in a backwards compatible way with openapi, so probably better to wait until it's really needed and then give them a new name :)

Reply:
Maybe it's time we started thinking about v3 API :thinking: :smile:

Reply:
And then we can recycle upload_options, and have multiple targets. If upload_targets isn't defined we can just do what we do now and take upload_options straight from the compose request.

Reply:
I like it.  Let's do this.  I think multiple targets wont be an issue but we're about to find out.